campaigns/inferno_steadwicks_fall.tex

```diff
@@ -0,0 +1,246 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Inferno Campaign -- Dungeons and Devils}{2. Steadwick's Fall}{\images/firewall.png}
```

Comment:
Those numbers (`2.` in this case) don't look so good in the ToC. Maybe for campaigns it should be a custom argument?